index.template.html

```diff
@@ -5239,8 +5239,13 @@ <h3>Extension and Import</h3>
     </pre>
 
     <p>
+        It is mentioned that the <code>tm:ref</code> or <code>tm:extends</code> features also allows transitive extension (a hierarchy of extensions). For example, assuming there are 3 TMs 
+        <code>ColorableLamp</code> which defines a <code>tm:extends</code> of the TM <code>DimmableLamp</code> which itself defines a <code>tm:extends</code> of the TM <code>SwitchableLamp</code>.
+        Consequently, the <code>ColorableLamp</code> TM also extend all definitions of both <code>DimmableLamp</code> and <code>SwitchableLamp</code>. It should be ensured that no recursive extensions are defined 
```

Comment:
Shouldn't the formulation "It should be ensured that no recursive extensions are defined" be instead:
"Recursive extensions leading to an infinite loop MUST NOT be defined."?